EightfoldAI/octuple
# a11y: Added focus border on Tabs.

## SUMMARY:

- Added focus border on Tabs.

## GITHUB ISSUE (Open Source Contributors)

## JIRA TASK (Eightfold Employees Only):
Jira task - https://eightfoldai.atlassian.net/browse/ENG-117818

## CHANGE TYPE:


## TEST COVERAGE:


## TEST PLAN:

- pull branch down locally and start storybook (yarn storybook)
- Navigate to the tabs component, and using tab focus on any of the tabs and observe that a focus border is visible.
